Podfile.lock

```diff
@@ -311,6 +321,6 @@ SPEC CHECKSUMS:
   WalletConnectSwift: 046ae74e8cb69445be9c94c66ef66f9cb019b68f
   web3swift: 06118d4c4edc801444aaa995bbbddeda176b97ef
 
-PODFILE CHECKSUM: eed0b6581ff81af0f01f5be1e1134299396facb9
+PODFILE CHECKSUM: ff6d53cc41e2fbd275aa69e2b1f17e8bad6530c8
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```

Comment:
Need to run `bundle` and `bundle exec pod install` first to get the correct gem version.